ccan/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 publish_internal_headers(ccan
+  align.h
```

Comment:
It it wasn't downloaded from ccan it doesn't belong in this directory, put it in util instead.

Reply:
OH I'm dumb, I just realized ccan is a different repository (though it's just a directory for common code).
I'll see if there's anything existing we can use, if not I will move to util.